tests/testthat/test-connect_lamindata.R

```diff
@@ -0,0 +1,58 @@
+test_that("Connecting to lamindata works", {
+  temp_lamin_dir <- tempfile()
+  temp_lamin_dir2 <- file.path(temp_lamin_dir, ".lamin")
+  dir.create(temp_lamin_dir2, recursive = TRUE, showWarnings = FALSE)
```

Comment:
Could you just use `tempdir()` here?

Reply:
No -- because `tempdir()` is a temporary directory that is created once per R session, so calling `tempdir()` multiple times results in the same result.